src/js/core/widget/core/Appbar.js

```diff
@@ -351,6 +368,51 @@
 				expandedTitle.style.opacity = expandLevel;
 			};
 
+			/**
+			 * Expand AppBar
+			 * @method expand
+			 * @member ns.widget.core.Appbar
+			 */
+			prototype.expand = function () {
+				var self = this;
+
+				self.element.style.height = "";
+				self.element.classList.add(classes.expanded);
+				self._appbarState = states.EXPANDED;
+				self._setTitlesOpacity(1);
+				utilsEvents.trigger(self.element, "appbarexpanded");
+			};
+
+			/**
+			 * Animation transition end event handler
+			 * @method _onTransitionEnd
+			 * @member ns.widget.core.Appbar
+			 * @protected
+			 */
+			prototype._onTransitionEnd = function () {
+				ns.engine.getBinding(this._ui.page, "Page").refresh();
```

Comment:
I think it's worth adding comment why the page needs to be refreshed.

Reply:
Done.